src/gen_smtp_server_session.erl

```diff
@@ -546,7 +554,7 @@ handle_request({<<"MAIL">>, Args},
 									?log(debug, "error: ~s~n", [Message]),
 									send(State, Message),
 									{ok, State};
-								NewState ->
+								#state{envelope = Envelope} = NewState ->
```

Comment:
it's actually a small bugfix, because we used to loose the value of `#envelope.expectedsize` we set on lines 522 and 528